### ISSUE TITLE
[missing fetures] compared to CS3 there are some missing block-tags

### ISSUE THREAD

**Issue by MBelhardi** [cubex2/customstuff4#21]
It's missing, compared to custom stuff 3 the following block-tags:
	- harvesting: tool, level, and silktouch (yes/no)
	- is Burning: will the Block take damage by burning or not
	- placement: where can the blocks be placed (floor, ceiling and/or wall)

I don't know about the following block-tags, maybe it is possible, but I couldn't find how in the wiki:
	- define more than 1 drop per block, when harvested (I mean not the same drops, more so when I harvest, for example an tree, I get 1 plank, and 2 sticks)
	- define a dropchance (min and max)
something like so:
```
	"drop": 
	{
		{"item": "minecraft:coal", "metadata": "1", "amount": [1, 2]},
		{"item": "minecraft:stick", "amount": [3, 4]},
	}
```
which would give me, when I harvest an dead log, some charcoal, and some sticks...

**Comment by MBelhardi** [comment]
at the moment it seems, that the block drops itself, when harvested, even when defined other drops. sure, sometimes the last defined item will drop, but other items defined won't drop...
there should be an option to disable block-self-drop

**Comment by cubex2** [comment]
You can only define one item at the moment, so it's impossible that the block drops itself if you set the drop to another block.

**Comment by MBelhardi** [comment]
That exactly what I thought would happen, but than, as I tried to harvest my Block, the first 3 to 4 blocks that dropped were the block itself, and all the other was, as I said, the last defined item, here how I defined it:
```
	"drop": 
	{
		"0": {"item": "minecraft:coal", "metadata": 1, "amount": 1},
		"0": {"item": "minecraft:stick", "amount": 3}
	},
```
With this, I have tried to determine if it is possible to define several drops. There was no error, so I tried to harvest the block, with the result I've told.

**Comment by cubex2** [comment]
There is no error because the second line just overrides the first one as they have the same key, '0' here. There is no way the dropped block can change after harvesting a few blocks, something else have to happen there.

**Comment by MBelhardi** [comment]
That's why I brought this up here, I couldn't understand, why the block itself would drop, when I have defined other Items, that should have to drop. Also I couldn't understand why the drop changed. I mean, I did know, that when no charcoal dropped, that the first definition was overwritten by the second, but I can't think of any way, why the block itself could be droped. I even tried to harvest a woll of 4x2x4, every block of this wall dropped itself, not one time there was dropped the defined Item. when I instead harvest  a tree-like formation (many blocks over another), then the first 3 to 4 will drop the block itself, the rest will be the defined item. I even tried to take out all other mod's, except for CS4 and forge of course, but the result was the same.
